src/StaticLint.jl

```diff
@@ -18,7 +18,16 @@ const Symbol = SymbolServer.stdlibs[:Core][:Symbol]
 const Int = SymbolServer.stdlibs[:Core][:Int]
 const Float64 = SymbolServer.stdlibs[:Core][:Float64]
 const Vararg = SymbolServer.FakeTypeName(Core.Vararg)
-isva(x) = (x isa SymbolServer.FakeTypeName && x.name.name == :Vararg && x.name.parent isa SymbolServer.VarRef && x.name.parent.name == :Core) || (x isa SymbolServer.FakeUnionAll && isva(x.body))
+
+isva(x::SymbolServer.FakeUnionAll) = isva(x.body)
+if Vararg isa Type
```

Comment:
```suggestion
@static if Vararg isa Type
```
maybe?

Reply:
I had that at first, but it's in a baremodule, so I didn't bother with trying to explicitly import it.

Reply:
Ok, was easy enough, so I changed it